### DISCUSSION TITLE
yet another public amber python client

### DISCUSSION THREAD

**Discussion by PaulSorenson** [amberelectric/public-api#62] [Show and tell]
# Amber public python API client

I uploaded code I am using myself to collect and transform data from the Amber public API [here](https://github.com/PaulSorenson/amber).

It's currently setup to write to PostgreSQL/timescaledb and MQTT - that part is pluggable. I use the postgres data for grafana visualisations and the MQTT for Home Assistant.

If you want to run it you also need https://github.com/PaulSorenson/aioconveyor.

**Comment by madpilot** [comment]
Awesome!
